src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Cancellation.cs

```diff
@@ -89,6 +89,7 @@ private sealed class SetTcsContent : StreamContent
             protected override Task SerializeToStreamAsync(Stream stream, TransportContext context)
             {
                 _tcs.SetResult(true);
+                Ticks = Environment.TickCount64;
```

Comment:
Is the `TaskCompletionSource` used somewhere? We could probably get rid of it.
Otherwise, I'd at least swap setting of `Ticks` with signaling `_tcs` so that if there is someone waiting on it, it will have `Ticks` ready when unblocked.

Reply:
it is used in test above. That one is also flaky but I'm not ready to lump it to this change.